code/game/objects/items/stacks/stack.dm

```diff
@@ -231,6 +231,10 @@
 /obj/item/stack/proc/get_max_amount()
 	return max_amount
 
+var/to_transfer = 0
```

Comment:
Originally I put it in the typepath because I wanted /obj/item/stack/proc/get_amount_transferred() to be with the other getters and had no clue where to put it. But then Birdtalon said 

"Birdtalon (Character name): @MINIMAN10000 you can't just pull that car out of the proc and declare it on /object/item/stack"

So I figured alright if I shouldn't put it there I will put it immediately above get_amount_tranferred()

It's only purpose is to declare to_transfer for get_amount_transferred() to prevent compilation error.

I'm all ears if there's a better way to achieve this.

Reply:
This should be where you put it originally, it was fine there.

Reply:
Alright I wasn't sure if putting it under "/obj/item/stack" would cause some sort of problem, I'll go ahead and move it back.